docpages/advanced_reference/automating-with-jenkins.md

```diff
@@ -1,27 +1,26 @@
-\page automating-with-jenkins Automating your bot with Jenkins
+\page automating-with-jenkins Automating Your Bot with Jenkins
 
 \note This page does NOT go into explaining how to install Jenkins, nor how to initially setup Jenkins. This is a tutorial for the CMake version with Linux (more specifically Ubuntu 22.04 LTS). If you don't know how to use CMake or you don't use CMake for your bot (and would like to) then please visit \ref buildcmake. If you wish to automate this tutorial from GitHub pushes then you can simply download the GitHub plugin for Jenkins, set that up and this tutorial will still work as this tutorial will only build what it can see!
 
-### Getting started
+## Getting Started
 
-First of all, you'll want to create your project. For this, we'll use a "Freestyle project" as we're just going to be calling some bash commands to tell CMake to build. We'll be calling this "DiscordBot" but you can name it whatever you want (I would advise against non-ascii characters).
+First of all, you'll want to create your project. For this, we'll use a "Freestyle project" as we're just going to be calling some bash commands to tell CMake to build. We'll be calling this "DiscordBot" but you can name it whatever you want. (I would advise against non-ASCII characters.)
 
 \image html jenkinsproject.png
 
-From here, just hit `Ok` and now you've created your Jenkins project, Well done! From here you can add a description, change the security policy (if your jenkins is public), really whatever your heart desires.
+From here, just hit `Ok` and now you've created your Jenkins project. Well done! From here you can add a description, change the security policy (if your Jenkins is public), really whatever your heart desires.
 
-### Automating the building process.
+## Automating the Building Process
 
-Scrolling down, you'll find `Build Steps` (You can also click `Build Steps` on the left). Here, you'll want to hit `Add build step` and hit `Execute shell`.
+Scrolling down, you'll find `Build Steps`. (You can also click `Build Steps` on the left.) Here, you'll want to hit `Add build step` and hit `Execute shell`.
```

Comment:
I think the brackets can be removed here. With the additional punctuation added, it renders the brackets useless.

Reply:
the way we were shown as kids was that brackets dont need a full stop inside or outside. In this case removing the brackets is definitely nicer

Reply:
The capitalisation of You gave me the feeling these are meant to be separate sentences, I was considering this or making the you lowercase, this felt closer to original intent. Happy to drop the brackets.

Reply:
> the way we were shown as kids was that brackets dont need a full stop inside or outside. In this case removing the brackets is definitely nicer

Yeah this is what I learnt as a kid too. I then googled it and learnt you can put punctuation inside brackets, if the content in the brackets is a complete new sentence (though, yet to know if that's right for the uk lol)

Reply:
Well yeah, you don't put punctuation inside if it's just an insert into a sentence. If it forms its own sentence on the other hand...